docs/data/core-blocks.md

```diff
@@ -0,0 +1,86 @@
+# **core/blocks**: Block Types Data
+
+## Selectors 
+
+### getBlockType
+
+Returns a block type by name.
+
+*Parameters*
+
+ * state: Data state.
+ * name: Block type name.
+
+
+### getCategories
+
+Returns all the available categories.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Categories list.
+
+### getDefaultBlockName
+
+Returns the name of the default block name.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Default block name.
+
+### getFallbackBlockName
+
+Returns the name of the fallback block name.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Fallback block name.
+
+## Actions
+
+### addBlockTypes
+
+Returns an action object used in signalling that block types have been added.
+
+*Parameters*
+
+ * blockTypes: Block types received.
```

Comment:
Shouldn't this be `Block type(s) to add to the store`? This is an action that adds new blocks to the editor, right?